### PR TITLE
amass-db: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.es/common/amass-db.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# amass db
+
+> Interacción con una base de datos Amass.
+> Más información: <https://github.com/OWASP/Amass/blob/master/doc/user_guide.md#the-db-subcommand>.
+
+- Lista de todas las enumeraciones realizadas en la base de datos:
+
+`amass db -dir {{ruta/a/directorio_base_de_datos}} -list`
```
Comment (blueskyson) [inline pages.es/common/amass-db.md:8]:
```suggestion
`amass db -dir {{ruta/al/directorio_base_de_datos}} -list`
```

File: pages.es/common/amass-db.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Muestra resultados para un índice de enumeración y un nombre de dominio especificados:
+
+`amass db -dir {{ruta/a/directorio_base_de_datos}} -d {{nombre_dominio}} -enum {{indice_de_lista}} -show`
```
Comment (blueskyson) [inline pages.es/common/amass-db.md:12]:
```suggestion
`amass db -dir {{ruta/al/directorio_base_de_datos}} -d {{nombre_dominio}} -enum {{indice_de_lista}} -show`
```

File: pages.es/common/amass-db.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Lista todos los subdominios encontrados en un dominio dentro de una enumeración:
+
+`amass db -dir {{ruta/a/directorio_base_de_datos}} -d {{nombre_dominio}} -enum {{indice_de_lista}} -names`
```
Comment (blueskyson) [inline pages.es/common/amass-db.md:16]:
```suggestion
`amass db -dir {{ruta/al/directorio_base_de_datos}} -d {{nombre_dominio}} -enum {{indice_de_lista}} -names`
```

File: pages.es/common/amass-db.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Muestra un resumen de los subdominios encontrados dentro de una enumeración:
+
+`amass db -dir {{ruta/a/directorio_base_de_datos}} -d {{nombre_dominio}} -enum {{indice_de_lista}} -summary`
```
Comment (blueskyson) [inline pages.es/common/amass-db.md:20]:
```suggestion
`amass db -dir {{ruta/al/directorio_base_de_datos}} -d {{nombre_dominio}} -enum {{indice_de_lista}} -summary`
```

**Review by blueskyson — APPROVED** [review]

File: pages.es/common/amass-db.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# amass db
+
+> Interacción con una base de datos Amass.
```
Comment (navarroaxel) [inline pages.es/common/amass-db.md:3]: Closer translation.
```suggestion
> interactúa con una base de datos Amass.
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**